CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # KitchenSink 1.3.0 Release Notes (2017-08-25)
 
+## Changes since 1.3.0
```

Comment:
Should line 1 be

    # KitchenSink 1.3.1 Release Notes (2017-08-28)

... or even deleted?

Reply:
I would say deleted